CMakeLists.txt

```diff
@@ -272,8 +272,13 @@ target_link_libraries(
           http_parser
 )
 target_compile_definitions(cchost PRIVATE CCF_LOGGER_NO_DEPRECATE)
+set_property(TARGET cchost PROPERTY POSITION_INDEPENDENT_CODE ON)
```

Comment:
I think we have a host library wrapper, which probably needs to be renamed to add_static_ccf_library() or some such but is still a valuable place to centralise common build/link time options.

Reply:
Agreed, but I think worth reading separately, so listed in #7130.